pkgs/development/libraries/ffmpeg/generic.nix

```diff
@@ -68,9 +70,19 @@ let
   vaapiSupport = reqMin "0.6" && ((isLinux || isFreeBSD) && !isAarch32);
 
   vpxSupport = reqMin "0.6" && !isAarch32;
+
+  /* FIXME
+  libva' = if headlessBuild then libva-headless else libva;
+  x264' = if headlessBuild then x264-noffmpeg else x264;
+  */
+  libva' = libva;
+  x264' = x264;
+  openglSupport' = if headlessBuild then false else openglSupport;
+  sdlSupport' = if headlessBuild then false else sdlSupport;
+  pulseaudioSupport' = if headlessBuild then false else pulseaudioSupport;
```

Comment:
Why don't we move this into the none prime options? Also on a headless server you can still require pulseaudio support like on a music streaming box that powers an audio setup.